src/current/cockroachcloud/upgrade-to-v22.1.md

```diff
@@ -27,8 +27,11 @@ The upgrade process depends on the number of nodes in your cluster. Select wheth
 
 ## Step 3. Understand the upgrade process
 
-<section class="filter-content" markdown="1" data-scope="multi-node">
-In a multi-node cluster, the upgrade does not interrupt the cluster's overall health and availability. One node is stopped and restarted with the new version, then the next, and so on, pausing for a few minutes between each node. This "rolling upgrade" takes approximately 4-5 minutes per node and is enabled by CockroachDB's [multi-active availability](../{{site.current_cloud_version}}/multi-active-availability.html) design.
+In a multi-node cluster, the upgrade does not interrupt the cluster's overall health and availability. An admin stops one node at a time and restarts it with the new version, waits a few minutes to observe the upgraded node's behavior, then moves on to the next node. This "rolling upgrade" takes approximately 4-5 minutes per node and is enabled by CockroachDB's [multi-active availability](../{{site.current_cloud_version}}/multi-active-availability.html) design.
+
+{{site.data.alerts.callout_danger}}
+After beginning a major-version upgrade, Cockroach Labs recommends upgrading all nodes as quickly as possible. In a cluster with nodes running different major versions of CockroachDB, a query that is sent to an upgraded node can be distributed only among other upgraded nodes. Data accesses that would otherwise be local may become remote, and the performance of these queries can suffer.
```

Comment:
I don't think the user really has control over how quickly the nodes are upgraded in Dedicated. You basically trigger an upgrade and just observe the progress. So I guess I'm wondering what the user is expected to do w/r/t "upgrading all nodes as quickly as possible."

Reply:
I reverted adding the note but changed "An admin" to "Cockroach Cloud" for these pages.